pkg/cache/nodes.go

```diff
@@ -213,21 +212,22 @@ func (nc *schedulerNodes) updateNode(oldNode, newNode *v1.Node) {
 		return
```

Comment:
we must update the resources in the cachedNode, might need locking and a setter method:
`cachedNode.capacity = common.GetNodeResource(&newNode.Status)`

Reply:
Taken care here itself even though YUNIKORN-1091 has been filed for the same